### PR TITLE
[Behat] MCE: Friendly exception message if context is not tagged as context

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

File: src/Sylius/Behat/MultiContainerExtension/ContextRegistry.php [hunk 1]
```diff
@@ -40,7 +40,11 @@ public function add($serviceId, $serviceClass)
     public function getClass($serviceId)
     {
         if (!isset($this->registry[$serviceId])) {
-            throw new \InvalidArgumentException();
+            throw new \InvalidArgumentException(sprintf(
+                'Could not find class for service with id "%s". Make sure you tagged the context with "%s".',
+                $serviceId,
+                'sylus.behat.context'
```
Comment (pjedrzejewski) [inline src/Sylius/Behat/MultiContainerExtension/ContextRegistry.php]: What's the point of sprintfing it? :D

Comment (pamil) [inline src/Sylius/Behat/MultiContainerExtension/ContextRegistry.php]: I like it separated. Anyway, there's a typo! :D

Comment (pjedrzejewski) [inline src/Sylius/Behat/MultiContainerExtension/ContextRegistry.php]: Well, not a big deal for me, but still I think it is pointless. Not really more readable. I always assume that sprintf arguments are something dynamic.

**Comment by michalmarcinkowski** [comment]
:+1:

**Pull request merged**